gcc/common.opt

```diff
@@ -2266,6 +2266,11 @@ fsingle-precision-constant
 Common Report Var(flag_single_precision_constant) Optimization
 Convert floating point constants to single precision constants.
 
+fftz-math
+Common Report Var(flag_ftz_math) Optimization
+Treats as if denormal floating points are flushed to zero in floating
```

Comment:
This help text needs updating. 'Makes optimizations assume denormal floating point values must be flushed to zero in floating operations.' or similar. I think key here is that the optimizations do not assume 'ftz' is optional and its semantics do not have to be preserved.